vernac/egramcoq.ml

```diff
@@ -268,15 +268,15 @@ let custom_entry_locality = Summary.ref ~name:"LOCAL-CUSTOM-ENTRY" String.Set.em
 let create_custom_entry ~local s =
   if List.mem s ["constr";"pattern";"ident";"global";"binder";"bigint"] then
     user_err Pp.(quote (str s) ++ str " is a reserved entry name.");
-  let sc = "constr:"^s in
+  let sc = s in
```

Comment:
Here, I think I would keep a prefix to ensure that users can use any name they want w/o risking a collision. The prefix could be e.g. "custom:".

Reply:
I could do this, but I believe the Coq currently only keeps a map of grammar extension symbols, not of all grammar symbol.  It would be useful to keep a map of all entries, but currently `coqpp` allows nonterminal names that are local to an mlg and therefore may not be unique.  Your thought?

Reply:
> It would be useful to keep a map of all entries, but currently `coqpp` allows nonterminal names that are local to an mlg and therefore may not be unique.

Good question and good point.

Actually, the name is used only for printing as Camlp5 internally refers to entries using a pointer.

So, the question seems to be only about possible collision when calling `Print Grammar` and in the ouput of `Print Grammar`.

Reply:
I put in the "custom:" prefix.